ansible/roles/girder/tasks/main.yml

```diff
@@ -64,10 +64,9 @@
 
 - name: Clone ctk-cli
   git:
-    repo: git://github.com/commontk/ctk-cli.git
+    repo: git://github.com:cdeepakroy/ctk-cli.git
```

Comment:
I think this needs to be `git://github.com/cdeepakroy/ctk-cli.git` (slash, not colon after github.com).